llvm/include/llvm/IR/IRBuilder.h

```diff
@@ -2143,12 +2137,15 @@ class IRBuilderBase {
   }
 
   Value *CreateCast(Instruction::CastOps Op, Value *V, Type *DestTy,
-                    const Twine &Name = "") {
+                    const Twine &Name = "", bool IsNonNeg = false) {
```

Comment:
I think I'd rather not have this, given how the flag does not apply to all casts.

Reply:
ill revert that.